roman_psf_fp.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import sys
+from itertools import product
+
+logging.basicConfig(level=logging.INFO)
+
+bp_name = sys.argv[1]
+SCA = int(sys.argv[2])
+
+oversampling_factor = 8.0
+n_waves = None
+
+bp_dict = galsim.roman.getBandpasses()
+bp = bp_dict[bp_name]
+
+chromatic_psf = galsim.roman.getPSF(SCA=SCA, bandpass=bp_name, n_waves=n_waves, SCA_pos=galsim._PositionD(4088, 4088))
+
+step = 800
+size_list, e1_list, e2_list = [], [], []
+x_list, y_list = [], []
+aber_list = []
+for x, y in product(range(0, galsim.roman.n_pix, step), range(0, galsim.roman.n_pix, step)):
+  i, j = x//step, y//step
+  logging.info("Calculating PSF for i=%d, j=%d for SCA=%d", i, j, SCA)
+  chromatic_psf = galsim.roman.getPSF(SCA=SCA, bandpass=bp_name, n_waves=n_waves, SCA_pos=galsim._PositionD(x, y))
+  achromatic_psf = chromatic_psf.evaluateAtWavelength(bp.effective_wavelength)
+  aberrations = achromatic_psf._screen.aberrations
+  epsf = galsim.Convolve([achromatic_psf, galsim.Pixel(galsim.roman.pixel_scale)])
```

Comment:
`epsf` here stands for effective PSF, not empirical PSF. Apologies for the confusion.

Reply:
This convolves the PSF with the native pixel response and then draw as an image below without including the pixel response again. This should match what you get out of `roman_imsim` except that it doesn't include those pesky detector effects.